packages/shared/ExecutionEnvironment.js

```diff
@@ -8,7 +8,12 @@
  */
 
 export const canUseDOM: boolean = !!(
```

Comment:
I wonder if we should move the globals into this file, instead of having a separate file. Or move canUseDOM to the globals file?